check-requirements.in

```diff
@@ -1,7 +1,16 @@
 bandit
+darglint
 flake8
+flake8-annotations-complexity
+flake8-bugbear
+flake8-broken-line
+flake8-commas
 flake8-comprehensions
+flake8-expression-complexity
```

Comment:
Split up chunks of code that get too complex (max complexity left at the plugin's default).